src/ExecuteTimeWidget.ts

```diff
@@ -290,11 +291,13 @@ export default class ExecuteTimeWidget extends Widget {
       if (isLikelyAborted) {
         msg = '';
       } else if (endTime) {
-        if (
-          this._settings.minTime <=
-          differenceInMilliseconds(endTime, startTime) / 1000.0
-        ) {
+        const executionTimeMillis = differenceInMilliseconds(
+          endTime,
+          startTime
+        );
+        if (this._settings.minTime <= executionTimeMillis) {
```

Comment:
I think there is a bug here - minTime is in s and now this is compared in ms

Reply:
I added the conversion back in a new commit. Thanks!